src/pages/docs/creators/ray/cluster-yaml.md

```diff
@@ -72,6 +72,32 @@ Note that using `pip` requires its index URL to be present in [`outbound_urls`](
 initialization_commands: []
 ```
 
+### File mounts
+
+You can use file mounts to copy directories and files to the head node and the worker nodes.
+The file mounts are synced during the nodes' initialization, so they are available 
+when your application is executed.
+
+The property accepts both absolute and relative paths, both locally and remotely.
+Relative paths start in `/root/` on the providers, 
```

Comment:
probably depends on the specific image in use, right?

Reply:
it is not about the image workdir, it is about home of user we are sshing to (ie root)